packages/ubuntu_desktop_installer/lib/l10n/app_en.arb

```diff
@@ -441,4 +445,4 @@
   "supportedSoftware": "Supported software",
   "copyingFiles": "Copying files...",
   "installationFailed": "Installation failed"
-}
```

Comment:
The Flutter Intl extension for VS Code provides support for ARB files and insists on removing this line break. :)

Reply:
My Vim keeps adding it. Every tool, every choice. :smile: